docs/help/operator.md

```diff
@@ -1029,3 +1031,117 @@ stringData:
   jolokiaUser: alice
   jolokiaPassword: password1
 ```
+
+## Using Cert-Manager certificates to configure brokers
+
+Note: this feature currently is experimental. Feed back is welcomed.
+
+[cert-manager](https://cert-manager.io/) adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates.
+
+The operator provides options in the custom resource that utilizes cert-manager x509 certificates to configure SSL/TLS transports for brokers.
+
+For a certificate to be used by broker, it has to have the keystore configured in it(either `pkcs12` or `jks`). For example:
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: server-cert-pkcs12
+  namespace: my-cert
+spec:
+  commonName: "artemiscloud.io"
+  dnsNames:
+    - "artemis-broker-ss-0"
+  secretName: server-cert-pkcs12-secret
+  subject:
+    organizations:
+    - "www.artemiscloud.io"
+  keystores:
+    pkcs12:
+      create: true
+      passwordSecretRef:
+        name: keystore-password-secret
+        key: pkcs12-password
+  issuerRef:
+    name: test-selfsign-cluster-issuer
+    kind: ClusterIssuer
+```
+The above yaml represents a certificate named `server-cert-pkcs12` in namespace `my-cert`. The pkcs12 keystore is configured and its password is stored in a secret named `keystore-password-secret` and the key is `pkcs12-password`. This secret must exists in order for the certificate to be successfully created, and password secret should be in the same namespace as the certificate's.
+
+### Configuring SSL/TLS for management console
+
+Once you have the certificate ready you can configure the management console of the broker to used it:
+
+```yaml
+apiVersion: broker.amq.io/v1beta1
+kind: ActiveMQArtemis
+metadata:
+  name: artemis-broker
+spec:
+  console:
+    expose: true
+    sslEnabled: true
+    brokerCert: server-cert-pkcs12:my-cert
```

Comment:
is the order wrong here? namespace first? my-cert is the namespace at line 1050

Reply:
no, that's correct.